packages/strapi-plugin-graphql/services/schema-generator.js

```diff
@@ -5,7 +5,8 @@
  *
  * @description: A set of functions similar to controller's actions to avoid code duplication.
  */
-
+const { filterSchema } = require('@graphql-tools/utils');
+const { buildFederatedSchema, printSchema } = require('@apollo/federation');
```

Comment:
can you use printSchema from graphql instead.

Reply:
check